util/reggen/countermeasure.py

```diff
@@ -0,0 +1,80 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, List
+
+from .lib import check_keys, check_str, check_list
+
+# TO BE DEFINED
+CM_ASSETS = [
+    'FOO'
+]
+
+# TO BE DEFINED
+CM_TYPES = [
+    'BAR'
+]
+
+
+class CounterMeasure:
+    """Object holding details for one countermeasure within an IP block."""
+
+    def __init__(self, asset: str, cm_type: str, desc: str):
+        self.asset = asset
+        self.cm_type = cm_type
+        self.desc = desc
+
+    @staticmethod
+    def from_raw(what: str, raw: object) -> 'CounterMeasure':
+        """
+        Create a CounterMeasure object from a dict.
+
+        The dict must have the keys 'name' and 'desc', where 'name' has
+        to follow the canonical countermeasure naming convention.
```

Comment:
This statement describes the `raw` argument. Could you mention that? Also add description for `what` and mention that it is only used as debug aid. It took me a while to figure that out by squinting at the code :) Likewise below.

Reply:
As a note: this convention is used throughout reggen.

Reply:
Mentioned that this pertains to the `raw` argument.